src/tests/bulkConcepts/addBulkConcepts.test.jsx

```diff
@@ -51,4 +53,138 @@ describe('Add Bulk Concepts', () => {
+        checked: false,
+      },
+    };
+
+    const spy = jest.spyOn(wrapper.find('AddBulkConcepts').instance(), 'handleSelect');
+    wrapper.instance().forceUpdate();
+    wrapper.find('#ciel').at(0).simulate('click');
+    wrapper.find('.table-check').at(0).simulate('change', {
+      preventDefault: jest.fn(),
+      target: {
+        value: ['/users/nesh/collections/test/concepts/'],
+        checked: true,
+      },
+    });
+    wrapper.find('.table-check').at(0).simulate('change', event);
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('it calls hanldleBack', () => {
+    const props = {
+      fetchCielConcepts: jest.fn(),
+      addExistingBulkConcepts: jest.fn(),
+      cielConcepts: multipleConceptsMockStore.concepts.dictionaryConcepts,
+      isFetching: true,
+      match,
+    };
+    const wrapper = mount(<MemoryRouter>
+      <Provider store={store}>
+        <AddBulkConcepts {...props} />
+      </Provider>
+    </MemoryRouter>);
+    const spy = jest.spyOn(wrapper.find('AddBulkConcepts').instance(), 'handleBack');
+    wrapper.instance().forceUpdate();
+    wrapper.find('.btn.btn-secondary.btn-bulk-cancel').at(0).simulate('click');
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+
+  it('it calls hanldleAddAll and add a concept', () => {
+    const props = {
+      fetchCielConcepts: jest.fn(),
+      addExistingBulkConcepts: jest.fn(),
+      cielConcepts: multipleConceptsMockStore.concepts.dictionaryConcepts,
+      isFetching: true,
+      match,
+    };
+    const event = {
+      preventDefault: jest.fn(),
+      target: {
+        value: ['/users/nesh/collections/test/concepts/'],
+        checked: true,
+      },
+    };
+    const wrapper = mount(<MemoryRouter>
+      <Provider store={store}>
+        <AddBulkConcepts {...props} />
+      </Provider>
+    </MemoryRouter>);
+    const spy = jest.spyOn(wrapper.find('AddBulkConcepts').instance(), 'handleAddAll');
+    wrapper.instance().forceUpdate();
+    wrapper.find('#ciel').at(0).simulate('click');
+    wrapper.find('.table-check').at(0).simulate('change', event);
+    wrapper.find('.btn.btn-primary.btn-add-all').at(0).simulate('click');
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+
+  it('it calls hanldlePaginationClick', () => {
+    const props = {
+      fetchCielConcepts: jest.fn(),
+      addExistingBulkConcepts: jest.fn(),
+      cielConcepts: multipleConceptsMockStore.concepts.dictionaryConcepts,
+      isFetching: true,
+      match,
+    };
+    const wrapper = mount(<MemoryRouter>
+      <Provider store={store}>
+        <AddBulkConcepts {...props} />
+      </Provider>
+    </MemoryRouter>);
+    const spy = jest.spyOn(wrapper.find('AddBulkConcepts').instance(), 'handlePaginationClick');
+    wrapper.instance().forceUpdate();
+    wrapper.find('#ciel').at(0).simulate('click');
+    wrapper.find('.fas.fa-angle-double-right.nxt').at(0).simulate('click');
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
```

Comment:
I think you should initiate a beforeEach function in this file because there is a lot of repeated code as previously stated. However only use it in the test code blocks you have added.